app/src/main/java/com/example/bottomnav/Parse.java

```diff
@@ -20,8 +20,7 @@
 
 public class Parse {
     private HashMap<String, Optional<DataDecoder>> decoderRepo= new HashMap<>();
-    private HashMap<String, String> canNameToStruct = new HashMap<>();
-    private HashMap<Integer, String> canIdToName = new HashMap<>();
+    private HashMap<Integer, String> IdToName = new HashMap<>();
```

Comment:
```suggestion
    private HashMap<Integer, String> idToName = new HashMap<>();
```

Consistent variable naming style?